picoscope/picobase.py

```diff
@@ -1058,9 +1058,9 @@ def getAWGDeltaPhase(self, timeIncrement):
 
         """
         samplingFrequency = 1 / timeIncrement
-        deltaPhase = int(samplingFrequency / self.AWGDACFrequency *
+        deltaPhase = int(round(samplingFrequency / self.AWGDACFrequency *
```

Comment:
is this really critical to change? i feel like sometimes these design choices, or errors, are best just left...

Reply:
No it's not really critical. Agree with you, will just revert the last commit.